streampark-flink/streampark-flink-kubernetes-v2/streampark-flink-kubernetes-core/src/main/scala/org/apache/streampark/flink/kubernetes/v2/operator/CROperator.scala

```diff
@@ -58,6 +60,10 @@ object CROperator extends CROperator {
   /** Apply FlinkDeployment CR. */
   // noinspection DuplicatedCode
   override def applyDeployment(spec: FlinkDeploymentDef): IO[Throwable, Unit] = {
+    for {
+      _ <- validateDeployFlinkDeploymentCRD
```

Comment:
The effect of this code segment will actually be ignored, it should move to line 69

Reply:
Ok, thanks for reviewing the code